packages/mux-video/src/index.ts

```diff
@@ -180,6 +177,23 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
     }
   }
 
+  get preload() {
+    const val = this.getAttribute('preload') as HTMLMediaElement['preload'];
+    if (val === '') return 'auto';
```

Comment:
this may not be correct. For example, in Chrome, `document.createElement('video').preload` will return `metadata`

Reply:
yes, but if you set the attribute preload to an empty string that corresponds with the preload prop being `auto`

basically attrs `preload=""` and `preload="auto"` are the same 

removing will use the default for the browser

Reply:
you mean `super.preload` to get the native video element prop? 
it's a shame the attribute callback is not fired without configuring these getters and setters. that would be ideal

Reply:
er, yes, however it is to get the native element property.

Reply:
I think it's correct though, I don't just return only `super.preload` in the getter because I find since the preload property is overridden on mux-video it's best to derive it from the attribute if possible. only for the case when no preload is set we check the super.preload to get the browser default.

Reply:
Sounds good!